src/utilities/resources.ts

```diff
@@ -6,16 +6,16 @@ import { getIntervalInMs } from './time';
  */
 export function sampleProfiles(
   profiles: Profile[] | null,
-  planStartTimeYmd: string | null,
-  duration: string | null,
-  offset?: string,
+  startTimeYmd: string | null,
+  durationInterval: string | null,
+  offsetMs?: string,
```

Comment:
This should probably be called `offset`, and the const on 16 should be `offsetMs`.

Reply:
Agreed